src/features/classification/Classification.js

```diff
@@ -89,7 +91,15 @@ const Classification = ({
                     <p className="bdl-Classification-modifiedBy" data-testid="classification-modifiedby">
                         <FormattedMessage
                             {...messages.modifiedBy}
-                            values={{ modifiedAt: formattedModifiedAt, modifiedBy }}
+                            values={{
```

Comment:
style: I think for consistency might be good to follow the file's pattern of having conditions declared outside of the main jsx. So if we add the `importedBy` message I mentioned above, it could be something like:

```
const modifiedByMessage = isImportedClassification ? messages.importedBy : messages.modifiedBy;
```
And then:
```
<FormattedMessage {...modifiedByMessage} values={...} /> 
```